contrib/build-wheel.sh

```
@@ -88,21 +92,227 @@ if [ "$CUDA_MAJOR" -ne 12 ] && [ "$CUDA_MAJOR" -ne 13 ]; then
+
+    deactivate
+    # torch + nvidia-* in each venv is several GB; tear down so the docker
+    # layer does not get too large across the (python, torch) matrix.
+    rm -rf "$VENV_PATH"
+}
+
+repair_wheel() {
+    local IN_DIR=$1
+    local OUT_DIR=$2
+    mkdir -p "$OUT_DIR"
+    auditwheel repair $AUDITWHEEL_EXCLUDES "$IN_DIR"/nixl*.whl --plat $WHL_PLATFORM --wheel-dir "$OUT_DIR"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)